helm/templates/_statefulset.yaml

```diff
@@ -16,6 +16,7 @@
 - image: {{ .Values.goQuai.image.name -}}:{{- .Values.goQuai.image.version }}
   imagePullPolicy: Always
   name: prime-go-quai
+  command: ["./build/bin/go-quai", --$(NETWORK), "--slices", $(SLICES)", "--syncmode", "full", "--http", "--http.vhosts=*", "--ws", <MINER_ETHERBASE>, "--http.addr", "0.0.0.0", "--http.api", "eth,net,web3,quai,txpool,debug", "--ws.addr", "0.0.0.0", "--ws.api", "eth,net,web3,quai,txpool,debug", "--port", $(TCP_PORT), "--http.port", $(HTTP_PORT), "--ws.port", $(WS_PORT), "--ws.origins=*", "--http.corsdomain=*", "--gcmode", "archive", "--nonce", <NONCE>, <REGION>, <ZONE>, <SUB>, <DOM>]
```

Comment:
This looks like it might not work with prime where there is no DOM or zones where there is no SUB

because of the "," you have between the <VALUE>'s